### PR TITLE
feat: Add manila role

### PR DESCRIPTION
**Pull request by okozachenko1203**

### CODE REVIEW

File: roles/manila/vars/main.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      api: 3
+      data: 3
+      scheduler: 3
+      share: 3
+  conf:
+    manila:
+      oslo_messaging_notifications:
+        driver: noop
+
+  dependencies:
```
Comment (mnaser) [inline roles/manila/vars/main.yml:30]: Shouldn’t this live in the Helm chart?

File: roles/defaults/defaults/main.yml [hunk 2]
```diff
@@ -167,3 +167,8 @@ atmosphere_images:
   skopeo: quay.io/skopeo/stable:latest
   tempest_run_tests: us-docker.pkg.dev/vexxhost-infra/openstack/tempest:30.1.0-4
   vector: docker.io/timberio/vector:0.27.0-debian
+  manila_db_sync: quay.io/vexxhost/manila:zed
```
Comment (mnaser) [inline roles/defaults/defaults/main.yml:170]: Can you please put these in alphabetical order?

File: docs/developer/services.md [hunk 3]
```diff
@@ -37,7 +38,6 @@ list of services that are planned to be included in the future.
 
 - [Magnum](https://docs.openstack.org/magnum/latest/)
```
Comment (mnaser) [inline docs/developer/services.md:39]: Maybe remove magnum here tok

**Review by mnaser — CHANGES_REQUESTED** [review]
Small nuts.

File: roles/manila/vars/main.yml (hunk 1 above)
Comment (okozachenko1203) [reply roles/manila/vars/main.yml:30]: well, i didn't make this as default in the chart and just commented with description.

File: roles/defaults/defaults/main.yml (hunk 2 above)
Comment (okozachenko1203) [reply roles/defaults/defaults/main.yml:170]: ok

File: docs/developer/services.md (hunk 3 above)
Comment (okozachenko1203) [reply docs/developer/services.md:39]: ok

File: roles/manila/vars/main.yml (hunk 1 above)
Comment (okozachenko1203) [reply roles/manila/vars/main.yml:30]: because users can select other driver types or even they can use share service without driver

**Pull request merged**